internal/backend/db/dbgorm/sessions_test.go

```diff
@@ -68,7 +69,7 @@ func TestCreateSessionForeignKeys(t *testing.T) {
 
 	// negative test with non-existing assets
 	s := f.newSession(sdktypes.SessionStateTypeCompleted)
-	unexisting := "unexisting"
+	unexisting := uuid.New()
```

Comment:
in other tests u did something different. please choose either, but consistent way